wordpress-mobile/WordPress-iOS
# Use mobile secrets credentials for UI tests

This is another piece of the UI test cleanup for https://github.com/wordpress-mobile/WordPress-iOS/issues/11013.

Adds support for using mobile secrets credentials for our UI tests and updates the build script accordingly, using the same approach we're using for screenshot generation.

To test:

Try running the UI Login Tests (other tests currently fail, pending fixes). This will copy the `WPUITestCredentials` template into `WordPress/WordPressUITests/WPUITestCredentials.swift` and display errors due to the template placeholders.

Then, delete the `WordPress/WordPressUITests/WPUITestCredentials.swift` file. Using the template, create a file at `~/.mobile-secrets/iOS/WPiOS/WPUITestCredentials.swift` with your WordPress.com and self-hosted account details. Rerun the UI Login Tests and confirm your credentials file is copied and the login tests pass as expected.
